zcash_client_backend/src/data_api/testing.rs

```diff
@@ -1227,15 +1221,22 @@ impl<Cache, DbT: WalletRead + Reset> TestState<Cache, DbT, LocalNetwork> {
 
 /// Helper method for constructing a [`GreedyInputSelector`] with a
 /// [`standard::SingleOutputChangeStrategy`].
-pub fn input_selector<DbT: InputSource>(
+pub fn input_selector<DbT: InputSource>() -> GreedyInputSelector<DbT> {
+    GreedyInputSelector::<DbT>::new()
+}
```

Comment:
I cannot find any uses of this. Since we're changing its arguments and since `GreedyInputSelector::new()` is already public, why not just delete it?

In any case, if it is kept then the "with a [`standard::SingleOutputChangeStrategy`]" part of the comment no longer makes sense, because the input selector is now independent of the change strategy.

```suggestion
```